streamlitDemo/utility.py

```diff
@@ -547,7 +557,7 @@ def extract_audio_from_live_stream(urlManifest):
 
             if audioLineMatch:
                 # wait for the audio to be written to file before sending 
-                time.sleep(3)
+                time.sleep(2)
```

Comment:
reduce the live video delay a little with this change. more work will have to be done with https://github.com/nsmq-ai/nsmqai/issues/83